R/pkg/R/client.R

```diff
@@ -42,6 +42,19 @@ determineSparkSubmitBin <- function() {
   }
   sparkSubmitBinName
 }
+# R supports both file separator in the file path (Unix : / and Windows: \) irrespective of the operating system
+# but when passing file path to Java or script program, it has to be converted according to operating system
+
+determinefileSeperator <- function() {
+  if (.Platform$OS.type == "unix") {
```

Comment:
@HyukjinKwon, cool, thanks! Could you double-check if this issue exists on Windows ,and this PR is the right fix. What I am wondering is that why system2 can't handle path separator since system2 is meant to be portable.